packages/cherry-pick-bot/.c8rc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "exclude": [".mocharc.js", "build/test"]
```

Comment:
optional: if you set `lines`, `branches`, `functions`, and add `--check` after `c8`, you can enforce a minimum coverage threshold, this can be nice because it makes people always improve coverage going forward:

https://github.com/yargs/yargs/blob/main/.nycrc#L11

Reply:
I don't know what those options mean. This was generated from our bot templates.

Reply:
They will throw if you go below that threshold for coverage for any of those tracked values, e.g., `lines` covered.